src/components/Dashboard/TodoList.vue

```diff
@@ -1378,48 +1350,55 @@ export default class TodoList extends Mixins(
         }
         break
 
-      case FilingTypes.INCORPORATION_APPLICATION:
+      case FilingTypes.INCORPORATION_APPLICATION: {
```

Comment:
I made this a code block to pre-empt a lint warning caused by declaring a new variable on line 1355.